lib/lrama/lexer.rb

```diff
@@ -29,23 +29,19 @@ class Lexer
       %empty
       %code
     )
-    PARAMETERIZING_TOKENS = [
-      'option\(',
-      'nonempty_list\(',
-      'list\(',
-      'separated_nonempty_list\(',
-      'separated_list\(',
-    ]
 
     def initialize(text)
       @scanner = StringScanner.new(text)
       @head_column = @head = @scanner.pos
       @head_line = @line = 1
       @status = :initial
       @end_symbol = nil
+      @lex_stack = []
```

Comment:
No value is pushed to this instance variable, could you remove this line and the first line of `#next_token`?

Reply:
Oops sorry, I deleted these.